spec/lib/jobs_spec.rb

```diff
@@ -21,8 +39,24 @@
     end
   end
 
+  describe '.scrub_keyword(query)' do
+    context 'when the search phrase contains a job related keyword' do
+      it 'returns the query with out generic job keywords' do
+        expect(Jobs.scrub_keyword('Nursing jobs')).to eq('Nursing')
+      end
+
+      it 'return blank if its equal to one of these job term keywords job,employment,posting,position' do
```

Comment:
typos: should be "returns blank", and `it's` instead of `its`

This spec lists the keywords that should be scrubbed, but doesn't test them. I'd reword the spec description to make it more generic, and test the keywords:

```
it 'returns blank when the query is a generic job keyword' do
  %w[job employment posting position].each do |query|
      expect(Jobs.scrub_keyword(query)).to eq('')
  end
end


```

Reply:
good catch